src/__tests__/actions/profile/profile.test.js

```diff
@@ -0,0 +1,146 @@
+import Enzyme from "enzyme";
+import React from "react";
+import configureMockStore from "redux-mock-store";
+import EnzymeAdapter from "enzyme-adapter-react-16";
+import thunk from "redux-thunk";
+import fetchMock from "fetch-mock";
+import {
+  IS_UPDATING, PROFILE_LOADED, UPDATE_SUCCESS,
+} from "../../../actions/types";
+import { loadProfile, updateProfile } from "../../../actions/profile/profileActions";
+
+Enzyme.configure({ adapter: new EnzymeAdapter() });
+const middleware = [thunk];
+const mockStore = configureMockStore(middleware);
+const API_HOST_URL = process.env.BASE_URL;
+
+let username;
+let url;
+describe("Profile Action tests", () => {
+  beforeEach(() => {
+    username = localStorage.getItem("username00000");
+    url = `${API_HOST_URL}/profiles/${username}`;
+  });
+  afterEach(() => {
+    username = "";
+    url = "";
+  });
+
+  it("tests if can load  a user profile", () => {
+    fetchMock.getOnce(url, {
+      body: {
+        profile: {
+          image:
+            "https://m0yfsbokq47e8jxhhbos.jpg",
+          bio: "samyy",
+          username: "testname",
+        },
+      },
+      headers: { "content-type": "application/json" },
+    }, { overwriteRoutes: false });
+    const expectedActions = [{ type: "PROFILE_LOADING" }, {
+      payload: {
+        bio: "samyy",
+        image: "https://m0yfsbokq47e8jxhhbos.jpg",
+        username: "testname",
+      },
+      type: PROFILE_LOADED,
+    }];
+    const store = mockStore();
+    return store
+      .dispatch(loadProfile())
+      .then(() => expect(store.getActions())
+        .toEqual(expectedActions));
+  });
+
+
```

Comment:
@CryceTruly This blank line needs to be worked on as well.

Reply:
Thanks @joelethan ,am done working on this too.

Reply:
@CryceTruly Am seeing this is still there.